packages/mobile/locales/de-DE/accountScreen10.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
We should leave empty json files `{}` to avoid conflicts with Crowdin.

Reply:
And rename the folder from `de-DE` to `de`.

Reply:
Crowin creates the files if they don't exist, so just deleting them works too